content/applications/general/iot/iot_box.rst

```diff
@@ -0,0 +1,91 @@
+=======
+IoT box
+=======
+
+To start using an IoT box:
+
+#. Make sure you have a valid IoT box subscription in addition to your Odoo subscription.
+#. Connect your :doc:`devices <devices>` to the IoT box.
+#. :ref:`Connect the IoT box <iot/iot_box/network>` to the network.
+#. :doc:`Connect the IoT box to your Odoo database <connect>`.
+
+.. note::
+   Devices can also be connected after the IoT box is added to the network and/or connected to the
+   database; however, a reboot of the IoT box is required.
```

Comment:
might be required. For example screen must be plugged before the IoT boot to display anything. However printers can be detected at any time